src/viur_cli/conf.py

```diff
@@ -171,4 +181,16 @@ def migrate(self):
         self.save()
 
 
+def print_changelog_from_github(user, repo):
+    url = f"https://raw.githubusercontent.com/{user}/{repo}/main/CHANGELOG.md"
+    response = requests.get(url)
+    if response.status_code == 200:
+        changelog_lines = response.text.split("\n")[:20]
```

Comment:
Contains every release exact 20 changes?

Reply:
No, but considering, that people are not updating frequently, i would say the more the better.